docs/code/MlNetCookBook.md

```diff
@@ -285,11 +285,11 @@ Or by creating a data model for it:
 ```csharp
 private class AdultData
 {
-    [LoadColumn("0", "10"), ColumnName("Features")]
-    public float FeatureVector { get; }
+    [LoadColumn(0, 10), ColumnName("Features")]
+    public float FeatureVector { get; set;}
```

Comment:
Maybe just
```suggestion
    public float FeatureVector;
```
?

Reply:
I'll double-check. I believe it has to be a property, not a field.